client/src/components/App/index.js

```diff
@@ -1,4 +1,11 @@
 import React from 'react';
+import Button from '../Utils/Button/index';
 import './style.css';
 
-export default () => <h1>Hello World</h1>;
+export default () => {
```

Comment:
Don't update App/ component so the others won't have any conflicts.
Unless you're adding a router keep the App the same :)